src_assets/windows/misc/gamepad/install-gamepad.bat

```diff
@@ -1,6 +1,32 @@
 @echo off
 setlocal enabledelayedexpansion
 
+rem Check if a compatible version of ViGEmBus is already installed (1.17 or later)
+set Version=
+for /f "usebackq delims=" %%a in (`wmic product where "name='ViGEm Bus Driver' or name='Nefarius Virtual Gamepad Emulation Bus Driver'" get Version /format:Textvaluelist`) do (
```

Comment:
I have 1.22 and it worked with just `ViGEm Bus Driver` 🤔

Reply:
That's the new name. The other name is for the 1.17 installer.

Reply:
Should we update this as well?

https://github.com/cgutman/LB_Sunshine/blob/9c4bc29f4ff034569daad5900106ebce66e80645/src_assets/windows/misc/gamepad/uninstall-gamepad.bat#L4

Reply:
I think that's okay to leave because our installer will never install such an old ViGEmBus version with that name.

Reply:
It's run when we uninstall. To remove vigem during uninstall. Unless we're okay with leaving it.

Reply:
Right, but if it's a version old enough to have the old name, it wasn't us that installed it in the first place.

Reply:
I see what you're saying... Nefarius Virtual Gamepad Emulation Bus Driver is the OLD name.

I was thinking this was the NEW name because of the trademark issue.